keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandlerTests.kt

```diff
@@ -210,112 +213,203 @@ internal object SecurityGroupHandlerTests : JUnit5Minutests {
       resetVpc()
     }
 
-    context("a security group with no ingress rules") {
-      before {
-        orcaService.stub {
-          on {
-            orchestrate(any())
-          } doAnswer {
-            CompletableDeferred(TaskRefResponse("/tasks/${randomUUID()}"))
+    sequenceOf(
+      "create" to SecurityGroupHandler::create,
+      "update" to SecurityGroupHandler::update.partially3(DiffNode.newRootNode())
+    )
```

Comment:
I want to run all the tests contained in this loop for both `create` and `update` without copy 'n pasting all the code